### DISCUSSION TITLE
Support cuda 12.4 in conda

### DISCUSSION THREAD

**Discussion by hiyyg** [facebookresearch/faiss#3996] [General]
# Summary

# Platform

OS: 

Faiss version: 

Installed from:  

Faiss compilation options: 

Running on:
- [x] CPU
- [x] GPU

Interface: 
- [ ] C++
- [x] Python

# Reproduction instructions

**Comment by pankajsingh88** [comment]
I see that cuda version 12 is supported with cudatoolkit == '12.1.1' (conda yaml [here](https://github.com/facebookresearch/faiss/blob/efa18da947cee1c90a1b68ee3555ca3aed2ce0b5/conda/faiss-gpu/meta.yaml#L12))

Can you try with cudatoolkit 12.1.1 version?

**Reply by pankajsingh88** [reply]
confirming that cuda 12.4 is not supported yet. Converted the discussion back to https://github.com/facebookresearch/faiss/issues/3998. Please use this issue for tracking progress.
